tests/utils/exec/rosacli/cluster_service.go

```diff
@@ -249,12 +273,23 @@ func (c *clusterService) IsBYOVPCCluster(clusterID string) (bool, error) {
 	return false, nil
 }
 
+// Check if the cluster is byo vpc cluster
+// Timeout need to be minutes unit
+func (c *clusterService) WaitForClusterReady(clusterID string, timeout int) error {
```

Comment:
This one should be deleted or ?
It does not seem to do much or ?

Reply:
Good catch!. Removed